modelina-website/src/pages/api/functions/PythonGenerator.ts

```diff
@@ -17,9 +17,7 @@ export async function getPythonModels(
 
   try {
     const generator = new PythonGenerator(options);
-    const generatedModels = await generator.generateCompleteModels(input, {
-      packageName: 'asyncapi.models'
-    });
+    const generatedModels = await generator.generateCompleteModels(input);
```

Comment:
You have to have the options present otherwise build fails 🙂 
```suggestion
    const generatedModels = await generator.generateCompleteModels(input, {});
```